apps/admin-gui/src/app/users/pages/user-detail-page/user-overview/user-overview.component.ts

```diff
@@ -66,4 +101,22 @@ export class UserOverviewComponent implements OnInit {
         style: 'user-btn'
       });
   }
+
+  changeEmail() {
+    const config = getDefaultDialogConfig();
+    config.width = '350px';
+    config.data = { userId: this.userID };
+
+    const dialogRef = this.dialog.open(ChangeEmailDialogComponent, config);
+
+    dialogRef.afterClosed().subscribe((success) => {
+      if (success) {
+        this.attributeService.getUserAttributeByName(this.userID, Urns.USER_DEF_PREFERRED_MAIL).subscribe(email => {
+          this.preferredMail = email
+
+          this.mailDataSource = new MatTableDataSource<Attribute>([this.preferredMail]);
```

Comment:
what if the prefferedmail is null? would the gui act atleast somehow normal? or it would broke everything?